src/main/java/org/folio/fqm/service/LocalizationService.java

```diff
@@ -23,7 +23,9 @@ public class LocalizationService {
   private static final String ENTITY_TYPE_COLUMN_LABEL_TRANSLATION_TEMPLATE = "mod-fqm-manager.entityType.%s.%s";
   // refers to a property inside an objectType column inside the entity type, e.g. "City" inside "Address" column inside "Users"
   private static final String ENTITY_TYPE_COLUMN_NESTED_LABEL_TRANSLATION_TEMPLATE =
-    "mod-fqm-manager.entityType.%s.%s.%s";
+  "mod-fqm-manager.entityType.%s.%s.%s";
```

Comment:
Nit: It looks like the line above your change got its formatting a little mangled (it's no longer indented as much as the other values nearby). That could easily just be GitHub being GitHub, though

Reply:
Oops yep, sometimes copy/pasting a line does this in my editor. Thanks!